### PR TITLE
Fix for GCC / Linux build

### PR DESCRIPTION
**Pull request by miles200**
Fix GCC build issue on Linux

-Add MapServerEvents BitStream include in event_gen/main.cpp

### CODE REVIEW

File: Utilities/event_gen/main.cpp [hunk 1]
```diff
@@ -38,6 +38,10 @@ void generate_cpp(const std::vector<DefFile> &files,const QString &group_name,co
     tgt_file.open(QFile::WriteOnly);
     QTextStream tgt_stream(&tgt_file);
     tgt_stream << "// THIS FILE WAS GENERATED, DO NOT EDIT\n";
+    if(group_name=="MapServerEvents")
+    {
+        tgt_stream << "#include \"BitStream.h\"\n";
+    }
     for(const DefFile &d : files)
```
Comment (nemerle) [inline Utilities/event_gen/main.cpp]: Now the question becomes, do we need to fix the group_name check below as well :)

**Comment by broxen** [comment]
This may relate to the `register_MapEvents()` to `register_MapServerEvents()` you (Nem) helped me with in MapEventRegistry.cpp.

Maybe a bad merge left some changes behind?

**Comment by nemerle** [comment]
```cpp
    if(group_name=="MapEvents")
    {
        tgt_stream << "#include \"GameData/chardata_serializers.h\"\n";
        tgt_stream << "#include \"GameData/clientoptions_serializers.h\"\n";
        tgt_stream << "#include \"GameData/gui_serializers.h\"\n";
    }
```
This piece of code was meant to provide the mapserver's generated file with some additional headers it might've needed ( since noone complained, we could probably remove this whole `if` ) :)

**Comment by miles200** [comment]
> ```c++
>     if(group_name=="MapEvents")
>     {
>         tgt_stream << "#include \"GameData/chardata_serializers.h\"\n";
>         tgt_stream << "#include \"GameData/clientoptions_serializers.h\"\n";
>         tgt_stream << "#include \"GameData/gui_serializers.h\"\n";
>     }
> ```
> 
> This piece of code was meant to provide the mapserver's generated file with some additional headers it might've needed ( since noone complained, we could probably remove this whole `if` ) :)

I will remove these and give it a test run tomorrow

**Comment by miles200** [comment]
@nemerle @broxen Removing that if block the build passes. Any specific testing I should be doing here or is a passing build good enough for now?

**Comment by nemerle** [comment]
Nha, If the build passes we're good :)

**Comment by miles200** [comment]
Thanks, should be good to merge @broxen

**Pull request merged**